tests/legacy-cli/e2e/utils/project.ts

```diff
@@ -42,6 +42,26 @@ export async function prepareProjectForE2e(name) {
 
   await ng('generate', 'e2e', '--related-app-name', name);
 
+  // Initialize welenium webdrivers.
+  // Often fails the first time so attempt twice if necessary.
+  const webdriverCommand = exec.bind(
+    null,
+    'node',
+    'node_modules/protractor/bin/webdriver-manager',
+    'update',
+    '--standalone',
+    'false',
+    '--gecko',
+    'false',
+    '--versions.chrome',
+    '101.0.4951.41',
+  );
```

Comment:
Nit 
```suggestion
  const webdriverCommand = exec(
    'node_modules/protractor/bin/webdriver-manager',
    'update',
    '--standalone',
    'false',
    '--gecko',
    'false',
    '--versions.chrome',
    '101.0.4951.41',
  );
```

Reply:
Then repeat that twice? I did the `.bind` so I can execute it twice without having to repeat the args twice). Could also do `() => exec(...)` instead of `bind` which I guess might be the preferred style now (with es6/ts, unlike 10y ago)?

I'm running it twice because the `webdriver-manager` fails 100% of the time when first run locally, then passes the second time... so I ran it twice here as well otherwise I can't run the tests locally at all. Previously when this was run with the yarn postinstall I had to run yarn twice before I could run the tests, but now that the tests themselves do it I can't run them at all without running it twice here within the tests...